torch/_torch_docs.py

```diff
@@ -9216,6 +9216,13 @@ def merge_dicts(*dicts):
 
 The shape of the tensor is defined by the variable argument :attr:`size`.
 
+.. note::
```

Comment:
No need for this to be a note. It can be part of the main docs. You can also move the paragraph right after the definition of the real part.